kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/PendingProduce.java

```diff
@@ -126,6 +126,12 @@ public void publishMessages() {
             throw new RuntimeException(e);
         }
 
+        if (persistentTopic.isSystemTopic()) {
+            log.error("Not support produce message to system topic: {}", persistentTopic);
+            responseFuture.complete(new PartitionResponse(Errors.INVALID_TOPIC_EXCEPTION));
+            return;
+        }
```

Comment:
I noticed this case was not protected by test case. If there's something wrong with system topics, you can open a new PR to handle it.

Reply:
When turn on system topic, the KOP doesn't work. I turn off it in the test.